src/main/jbake/content/jsonp005.adoc

```diff
@@ -11,24 +11,24 @@ prev=jsonp004.html
 JSON in Jakarta EE RESTful Web Services
 ---------------------------------------
 
-This section explains how the Java API for JSON Processing is related to
+This section explains how the Jakartaa JSON Processing is related to
 other Jakarta EE packages that provide JSON support for RESTful web
 services. See link:jaxrs.html#GIEPU[Chapter 32, "Building RESTful Web
-Services with JAX-RS,"] for more information on RESTful web services.
+Services with Jakarta RESTful Web Services,"] for more information on RESTful web services.
 
-Jersey, the reference implementation for JAX-RS (JSR 311) included in
+Jersey, the Jakarta RESTful Web Services implementation included in
 GlassFish Server, provides support for binding JSON data from RESTful
-resource methods to Java objects using JAXB, as described in
-link:jaxrs-advanced007.html#GKKNJ[Using JAX-RS with JAXB] in
-link:jaxrs-advanced.html#GJJXE[Chapter 34, "JAX-RS: Advanced Topics and
-an Example"]. However, JSON support is not part of JAX-RS (JSR 311) or
-JAXB (JSR 222), so that procedure may not work for Jakarta EE
+resource methods to Java objects using Jakarta XML Binding, as described in
```

Comment:
Sigh.  JAXB is part of Java SE 8, and thus Jakarta XML Binding is not part of Jakarta EE 8.
All the references to JAXB should remain for now.  I know it's a bit confusing...

Reply:
Thankfully we still have somone who knows what term to use in each particular situation.